src/ray/raylet/node_manager.cc

```diff
@@ -510,6 +516,13 @@ void NodeManager::HandleRequestObjectSpillage(
       });
 }
 
+void NodeManager::HandleRestoreSpilledObject(
+    const rpc::RestoreSpilledObjectRequest &request,
+    rpc::RestoreSpilledObjectReply *reply, rpc::SendReplyCallback send_reply_callback) {
+  local_object_manager_.AsyncRestoreSpilledObject(
+      ObjectID::FromBinary(request.object_id()), self_node_id_, nullptr);
```

Comment:
Don't we need to call send_reply_callback? Should that be the callback passed into AsyncRestore?

Reply:
Ah, I forgot about it. I think we can reply just right away? (since the object location change will be notified by pubsub anyway).

Reply:
I just moved it to callback.